cmd/minishift/cmd/start.go

```diff
@@ -422,13 +393,9 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Bool(metrics, false, "Install metrics (experimental)")
 	clusterUpFlagSet.Bool(logging, false, "Install logging (experimental)")
 	clusterUpFlagSet.String(openshiftVersion, version.GetOpenShiftVersion(), fmt.Sprintf("The OpenShift version to run, eg. %s", version.GetOpenShiftVersion()))
-}
-
-// initProxyFlags create the CLI flags which needs to be passed for proxy
-func initProxyFlags() {
-	proxyFlagSet.String(httpProxy, "", "HTTP proxy for virtual machine (In the format of http://<username>:<password>@<proxy_host>:<proxy_port>)")
-	proxyFlagSet.String(httpsProxy, "", "HTTPS proxy for virtual machine (In the format of https://<username>:<password>@<proxy_host>:<proxy_port>)")
-	proxyFlagSet.String(noProxyList, "", "List of hosts or subnets for which proxy should not be used.")
+	clusterUpFlagSet.String(httpProxy, "", "HTTP proxy used for downloading artefact and configure Docker as well as OpenShift (In the format of http://<username>:<password>@<proxy_host>:<proxy_port>). Overrides a potential HTTP_PROXY setting in the enviroment.")
+	clusterUpFlagSet.String(httpsProxy, "", "HTTPS proxy used for downloading artefact and configure Docker as well as OpenShift (In the format of https://<username>:<password>@<proxy_host>:<proxy_port>). Overrides a potential HTTPS_PROXY setting in the enviroment.")
+	clusterUpFlagSet.String(noProxyList, "", "List of hosts or subnets for which no proxy should be used.")
```

Comment:
These are not entirely for cluster up as we use it in subscription manager. Is that the reason you changed it to clusterUpFlagSet